### ISSUE TITLE
Splitting up Functions and Scripts as resources for others (Issue #26)

### ISSUE THREAD

**Issue by Sembrani** [loopuleasa/Dota2-TheCore-Config-Engine#55]
I'm happy to start working on this. I'll be able to split it up with a bit of time, and make it neater / more readable, but I won't start on the tutorials for now, leaving them for someone else, or for later.

**Comment by loopuleasa** [comment]
I can handle the tutorials if you make the modular bits! Just make sure it's the minimal things you need.

I can always format some readme files and make some relevant images/videos.

**Comment by Sembrani** [comment]
Cheers. Should have most of it done within a week.

**Comment by loopuleasa** [comment]
Note of a thing with + and - key press key depress functions. You have to have some forced "-keyDepressed" in some files to prevent cases like when you pressed ALT+SPACE then you depress only one of them and so on.

I had issues with +chatWheel and +scoreScreen in the past, and I had to add -chatWheel and -scoreScreen in the default binds so that it doesn't get stuck on screen. Haven't looked if you caught that case, but you should be wary.

**Comment by Sembrani** [comment]
Yeah, I understand the pitfalls behind the + and - functions. I think I have a pretty good grasp on why it happens like that, so I'm not too worried about forgetting about.

**Comment by Sembrani** [comment]
This is pretty much ready to put a pull request in for now. Will put a request in once the per-key quickcast is merged.

**Comment by Sembrani** [comment]
Can be closed now.

**Comment by loopuleasa** [comment]
Yep.

I'll handle making tutorials now and probably releasing the new 5.2 version soon + reddit advertisement.

You've done a lot of good work, mate. Be proud.

I would like it if you could fix the toggle health segmentation for reborn (which uses the defunct toggle command) and maybe add it to the LIBRARY as well.

In the meanwhile, I'll just mess with the readme tutorials.
